starknet_devnet/blueprints/rpc/misc.py

```diff
@@ -64,6 +67,9 @@ async def syncing() -> Union[dict, bool]:
 
 
 # pylint: disable=too-many-arguments
+# Events response does not currently conform to RPC specs
+# and will need fixing before validation is added
+# @validate_schema("getEvents")
```

Comment:
I discovered this while working on schema validation, but I'd prefer this is addressed in the separate issue.